cinema/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Consider adding test cases to this file to verify the functionality of your `Movie` model and any views or serializers you have implemented. This will help ensure the reliability and correctness of your application.